### PR TITLE
Change the wording to have the user reload from VACOLS.

### PR DESCRIPTION
**Pull request by paultag**
cc @KHarshawat @maryannbrody 

Something really silly happens where we lose our place after IAM gives us data.

This wording will at least get the user to work around the bug if it happens in prod. We've been super unable to duplicate this bug, but it surely exists. Hopefully this can help a little.

The current wording is super unhelpful. @maryannbrody do you wording that would work best here?

Related to https://github.com/department-of-veterans-affairs/caseflow/issues/577 - but I'm going to leave that bug open, since this doesn't solve the underlying issue.

### CODE REVIEW

File: app/views/web/404.html.erb [hunk 1]
```diff
@@ -3,7 +3,12 @@
 <% content_for :content do %>
 <div class="cf-app-msg-screen cf-app-segment cf-app-segment--alt">
   <h1 class="cf-msg-screen-heading">Page not found</h1>
-  <h2 class="cf-msg-screen-deck">Oops! It looks like the page you're looking for does not exist</h2>
-  <p class="cf-msg-screen-text">You might want to double check the URL for errors.</p>
+  <h2 class="cf-msg-screen-deck">
+      Oops! We can't find the correct page. Please reload Caseflow from VACOLS.
```
Comment (paultag) [inline app/views/web/404.html.erb]: Some comment here

Comment (KHarshawat) [inline app/views/web/404.html.erb]: @paultag 

`Oops! We can't find the correct page. Please close this window and relaunching Caseflow from VACOLS.`

File: app/views/web/404.html.erb [hunk 2]
```diff
@@ -3,7 +3,12 @@
...
+  </h2>
+  <p class="cf-msg-screen-text">
+      If you continue to get this page, please send feedback to us, so we
+      can help diagnose the error!
+  </p>
```
Comment (KHarshawat) [inline app/views/web/404.html.erb]: @paultag 

`If you continue to see this page, please contact the help desk.`

**Comment by paultag** [comment]
@KHarshawat wording changes made!

**Comment by shanear** [comment]
LGTM

**Pull request merged**